GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -516,14 +514,7 @@ private void SshBrowseClick(object sender, EventArgs e)
 
         private void LoadSshKeyClick(object sender, EventArgs e)
         {
-            if (string.IsNullOrEmpty(PuttySshKey.Text))
-            {
-                MessageBox.Show(this, _errorNoKeyEntered.Text, TranslatedStrings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
-            }
-            else
-            {
-                GitModule.StartPageantWithKey(PuttySshKey.Text);
-            }
+            PuttyHelpers.StartPageantIfConfigured(() => PuttySshKey.Text);
```

Comment:
Yes, it has to be a delegate because one of the callsites (I can't remember which one right now) can be invoked _before_ we have `Module` initialised, and this end up with a NRE.